packages/ui/README.md

```diff
@@ -0,0 +1,48 @@
+# Svelte + TS + Vite
```

Comment:
can we remove this and then also add some of the development style patterns (ie. capitalizate function names if they implement a domain interface)

Reply:
or dont even need to remove this, mainly just add the development patterns for new devs